org.gramps_project.Gramps.yml

```diff
@@ -0,0 +1,115 @@
+app-id: org.gramps_project.Gramps
+# flatpak-builder will not take com.gramps-project.Gramps
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: /app/bin/gramps
+rename-icon: gramps
+finish-args:
+# needs home access to access documents and pictures
+  - --filesystem=home
+# needs to own upstream name
+  - --own-name=org.gramps-project.Gramps
+# for gui
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+#needs network access for maps
+  - --share=network
+modules:
+  - name: BSDDB3
+    subdir: dist
+    builddir: true
+    config-opts:
+      - --enable-compat185
+      - --enable-shared
+      - --enable-cxx
+      - --enable-dbm
+      - --enable-stl
+    sources:
+      - type: archive
+        url: https://download.oracle.com/berkeley-db/db-5.3.28.tar.gz
+        sha256: e0a992d740709892e81f9d93f06daf305cf73fb81b545afe72478043172c3628
+      - type: patch
+        path: atomic.patch
```

Comment:
You should add here in the sources this entry:
```yaml
- type: shell
  commands:
    - cp -p /usr/share/automake-*/config.{sub,guess} .
```

This should fix the build on `aarch64`

Reply:
Thank you for the help.